cmd/osv-scanner/fixtures/encoding-integration-test-locks/UTF-8/requirements.txt

```diff
@@ -1,7 +1,29 @@
+#
```

Comment:
Can you have 2 requirements.txt files in that test, one manually generated with multiple file links (usage of the -r flag) and one automatically generated from a requirements.in ?

Reply:
good call, added another file (requirements-test.txt) that is not autogenerated with a link to another requirements file